src/CommonTestBase.sol

```diff
@@ -8,6 +8,7 @@ import {IERC20} from 'erc20-helpers/interfaces/IERC20.sol';
 library ChainIds {
     uint256 internal constant MAINNET = 1;
     uint256 internal constant OPTIMISM = 10;
+    uint256 internal constant GNOSIS = 100;
```

Comment:
This whole file needs to be refactored into PL formatting style. Want to just leave as-is for now and can do the whole file at once instead of piece-wise.